### PR TITLE
add support for transitioning Jira issues

### PR DESCRIPTION
**Pull request by pingunaut**
This PR enabled JIRA integration to transition Jira issues, i.e. Resolve, Close, Reopen...

### CODE REVIEW

File: Integrations/integration-jira.yml [hunk 1]
```diff
@@ -229,7 +230,7 @@ script:
             mdObj['description'] = dq(element, 'fields.description');
             mdObj['duedate'] = dq(element, 'fields.duedate');
             mdObj['ticket_link'] = dq(element, 'self');
-            mdObj['creted'] = dq(element, 'fields.created');
+            mdObj['created'] = dq(element, 'fields.created');
```
Comment (pingunaut) [inline Integrations/integration-jira.yml]: Fixed typo here.. maybe would have been better to put this into an own commit... please give a short feedback if this should be reverted and added to another PR

**Comment by pingunaut** [comment]
Any news on that?
As far as I can see, this functionality is included in #1705 right?
If yes, PR can be closed...

**Comment by dorsha** [comment]
@saharsabban / @anara123 ?

**Comment by saharsabban** [comment]
Looks like a typo from previous versions, the rest already merged.

**Comment by saharsabban** [comment]
Closing this PR while we already have this fix - will open new one for typo fix
